### PR TITLE
5

### PR DESCRIPTION
**Pull request by schoeyy**

### CODE REVIEW

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Tom, I'm afraid the app doesn't work or load. A problem with cors, and various errors - I'll send you a screenshot.

Looking at your backend repo I don't think CORS is setup right, you have it pointing to localhost why is probably why it's working for you, and not me. 

Please review these notes on exactly how to implement CORS: https://notes.northcoders.com/courses/js-front-end/react-projects

Once this is fixed, and you've tested it, then please re-submit this pull request. For clarity - I've reviewed no functionality or code at this stage.

File: nc-news/src/Article-Container.jsx [hunk 1]
```diff
@@ -39,24 +39,22 @@ const ArticleList = () => {
...
+            <img
+              className="article-image"
+              src={article.article_img_url}
+              alt={article.title}
+            />
+            <div className="article-text">
+              <h2 className="article-title">{article.title}</h2>
+              <p className="article-author">Author: {article.author}</p>
+              <p className="article-topic">Topic: {article.topic}</p>
+              <p className="article-create-date">Created at: {new Date(article.created_at).toLocaleDateString()}</p>
```
Comment (Jodiech) [inline nc-news/src/Article-Container.jsx:53]: Nice date formatting here 👍

File: nc-news/src/Article-Details.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      } catch (error) {
+        console.error("Error:", error);
+        setErrorMessage(error);
+      }
+    };
+
+    fetchArticle();
+  }, [id]);
+
+//   useEffect(() => {
```
Comment (Jodiech) [inline nc-news/src/Article-Details.jsx:33]: Please remove any comments in your code, keep it nice and tidy 😄

File: nc-news/src/Article-Details.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+    // finish add comment function
+
+    setNewComment("");
+  };
+
+  if (isLoading) {
+    return (
+      <div className="loading-spinner-container">
```
Comment (Jodiech) [inline nc-news/src/Article-Details.jsx:65]: Love the loading spinner 😎

**Review by Jodiech — APPROVED** [review]
Hey 👋 nice work on this ticket! Just remember to remove any comments before submitting a PR. I like your navigation with the link to go back to the home page at the bottom of the single article 👍  Nice work! Happy for you to merge 🤸

**Pull request merged**